core/embed/rust/src/ui/model_tt/component/error.rs

```diff
@@ -62,7 +65,7 @@ impl<T: AsRef<str>> Component for ErrorScreen<'_, T> {
         );
         self.message.place(message_area);
 
-        let (_, bottom_area) = ResultFooter::<T>::split_bounds();
+        let (_, bottom_area) = ResultFooter::<'a>::split_bounds();
```

Comment:
...and also here.

In general, `impl<'a> Component` should not be necessary unless `type Msg` depends on `'a`. It should always be `impl Component for X<'_>`.

Reply:
d0243af2e2ecd705cf927cfa3220d61d107fc09b